src/test/java/seedu/guest/storage/JsonGuestBookStorageTest.java

```diff
@@ -42,26 +42,26 @@ private Path addToTestDataPathIfNotNull(String prefsFileInTestDataFolder) {
 
     @Test
     public void read_missingFile_emptyResult() throws Exception {
-        assertFalse(readAddressBook("NonExistentFile.json").isPresent());
+        assertFalse(readGuestBook("NonExistentFile.json").isPresent());
     }
 
     @Test
     public void read_notJsonFormat_exceptionThrown() {
-        assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatGuestBook.json"));
+        assertThrows(DataConversionException.class, () -> readGuestBook("notJsonFormatGuestBook.json"));
     }
 
     @Test
-    public void readAddressBook_invalidPersonAddressBook_throwDataConversionException() {
-        assertThrows(DataConversionException.class, () -> readAddressBook("invalidGuestGuestBook.json"));
+    public void readGuestBook_invalidGuestBook_throwDataConversionException() {
+        assertThrows(DataConversionException.class, () -> readGuestBook("invalidGuestGuestBook.json"));
     }
 
     @Test
-    public void readAddressBook_invalidAndValidPersonAddressBook_throwDataConversionException() {
-        assertThrows(DataConversionException.class, () -> readAddressBook("invalidAndValidGuestGuestBook.json"));
+    public void readGuestBook_invalidAndValidGuestBook_throwDataConversionException() {
+        assertThrows(DataConversionException.class, () -> readGuestBook("invalidAndValidGuestGuestBook.json"));
     }
 
     @Test
-    public void readAndSaveAddressBook_allInOrder_success() throws Exception {
+    public void readAndSaveGuestBook_allInOrder_success() throws Exception {
         Path filePath = testFolder.resolve("TempAddressBook.json");
```

Comment:
Should the json File name be TempGuestBook.json? For this I am not too sure as well since I am not sure where the JSON File is located at, so I am just asking to double confirm

Reply:
fixed